src/ekobox/evol_pdf.py

```diff
@@ -93,8 +97,6 @@ def evolve_pdfs(
         info_update=info_update,
     )
 
-    # in the eko scales are squared
-    q2block_per_nf = {nf: np.power(q2s, 2) for nf, q2s in q2block_per_nf.items()}
```

Comment:
this removal is a very suspicious change ... because I introduced it in #391 (d5f63fe7ac995fd9d0d38dba48f2a7969de01c28 to be specific) and [unit tests were passing](https://github.com/NNPDF/eko/actions/runs/9956298042/job/27505888734#step:11:72) - but I have to remove it here to make [the unit tests pass](https://github.com/NNPDF/eko/actions/runs/10301208359/job/28512239152?pr=291#step:11:73) and I can not see how the other changes here would change that ...